fsharp-backend/tests/testfiles/int.tests

```diff
@@ -10,6 +10,10 @@
 //Int.negate_v0 -5 = 5
 //Int.negate_v0 5 = -5
 //Int.remainder_v0 5 0 = Error "`divisor` must be non-zero"
+Int.power_v0 8 5 = 32768
+Int.power_v0 0 1 = 0
+Int.power_v0 1 0 = 1
+Int.power_v0 1000 0 = 1
```

Comment:
Can you add a test with a negative number?

Reply:
I added the test case with a negative base. Should we add an exception for a negative exponent?
The number must be greater than or equal to zero. (Parameter 'exponent')

Reply:
We should catch the exception for sure, we don't want to throw in Dark StdLib code. What does the current Dark code in OCaml do? (You can check this by putting the code into the editor on darklang.localhost:8000/a/dark)